source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst

```diff
@@ -0,0 +1,38 @@
+.. _K8SPXC-1.4.0:
+
+================================================================================
+*Percona Kubernetes Operator for PXC* 1.4.0
+================================================================================
+
+:Date: March XX, 2020
+
+:Installation: `Installing Percona Kubernetes Operator for PXC <https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html#installation>`_
+
+New Features
+================================================================================
+
+* :jirabug:`K8SPXC-125`: Percona XtraDB Cluster 8.0 is now supported
+
+Improvements
+================================================================================
+
```

Comment:
please add K8SPXC-100

Reply:
Done, but I'm still not sure about the description. 
1) Is it PXC 8.0 only? 
2) Should this feature also be mentioned in the graceful shutdown or crash recovery doc?

Reply:
it is for both 5.7 and 9.0, the description is ok